public/index.html

```diff
@@ -24,10 +24,22 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
-  </head>
-  <body>
-    <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="root"></div>
-  </body>
+        <link
```

Comment:
Why were added two libraries of different versions of bootstrap?

Reply:
Good catch, one of the version was removed